core/templates/pages/exploration-player-page/services/audio-translation-manager.service.spec.ts

```diff
@@ -117,4 +117,30 @@ describe('Audio translation manager service', () => {
         })
       });
     });
+  it('should properly get html for Autogenerated Audio',
+    () => {
+      atms.setContentAudioTranslations(
+        testAudioTranslations, 'contentHtmlForAutogeneratedAudio', '');
+      expect(atms.getCurrentHtmlForAutogeneratedAudio()).toEqual(
+        'contentHtmlForAutogeneratedAudio'
```

Comment:
Can we add some representative HTML atleast? Just to make sure all the tags go through correctly

Reply:
Done